include/drivers/st25r3911b_nfca.h

```diff
@@ -0,0 +1,308 @@
+/*
```

Comment:
I think we can better organize st25r3911b related headers by adding an additional directory for them. This way, we won't pollute drivers directory with 4 st25r3911b related header files.

My suggestion:
drivers/st25r3911b_nfca.h -> drivers/st25r3911b/nfca.h
...

A similar thing is already done for source files. However, I would remove the name duplication from the path:
st25r3911b/st25r3911b_nfca.c -> st25r3911b/nfca.c

Reply:
Ok, changed